### DISCUSSION TITLE
UX > utilité de toujours écrire .nc sur l'écran

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#92] [Q&A]
Ca prend de la place... je me demande si on pourrait à pas à la aplacer poser un genre de label ?

**Comment by adriens** [comment]
Z'en pensez quoi  @lschaeffer313 @HakumenNC :grey_question:

**Comment by HakumenNC** [comment]
👍🏽 Il pourra se transformer en dropdown pour l'issue #71
